crates/hyperswitch_connectors/src/connectors/archipel.rs

```diff
@@ -481,15 +485,19 @@ impl ConnectorIntegration<PSync, PaymentsSyncData, PaymentsResponseData> for Arc
         req: &PaymentsSyncRouterData,
         connectors: &Connectors,
     ) -> CustomResult<Option<Request>, errors::ConnectorError> {
+        let auth_details = archipel::ArchipelAuthType::try_from(&req.connector_auth_type)?;
         let url = &self.get_url(req, connectors)?;
         let headers = self.get_headers(req, connectors)?;
+        let body = self.get_request_body(req, connectors)?;
```

Comment:
Is this intentional: do you need a body to perform payments sync?

Reply:
we don't actually need it - removed it 🙌